content/blog/2018-06-08-unconf18_recap_3.md

```diff
@@ -0,0 +1,81 @@
+---
+slug: "unconf_recap_3"
+title: "Unconf18 projects 1: jobstatus, motifator, QcodeR, opencv, trackmd"
+date: 2018-06-08
+authors:
+  - name: Maëlle Salmon
+    url: http://www.masalmon.eu/
+categories: blog
+tags:
+- community
+- ropensci
+- meetings
+- unconf
+- unconf18
+---
+
+Following other [unconf18 recaps](https://ropensci.org/tags/unconf18/), here is an overview of five other projects. In the following weeks, a handful of unconf18 groups will share more thorough posts about their work.
```

Comment:
this link not ideal: `[unconf18 recaps](https://ropensci.org/tags/unconf18/)`
because in future will link to unconf announcement and every post, not just recaps

Could say: "For day 3 of project recaps from [this year's unconf](https://ropensci.org/blog/2018/06/05/unconf18/), here is an overview of the next five projects. Stay tuned for the last recap tomorrow.